src/annotator/config/settings.js

```diff
@@ -152,6 +152,14 @@ function settingsFrom(window_) {
       return jsonConfigs[name];
     }
 
+    if (options.defaultValue === null) {
+      return true;
```

Comment:
I'm not clear on why this is here, if I fetch a setting with `settings.hostPageSetting('someSetting', { defaultValue: null })` and the page does not set a value for `someSetting` then I would expect the return value of `hostPageSetting` to be null, no?